src/workflows-app/components/ImportGithub.js

```diff
@@ -21,12 +24,18 @@ const constraints = {
   },
 };
 
-const ImportGithub = ({ setLoading, signal, onDismiss, workspace }) => {
+const ImportGithub = ({ setLoading, signal, onDismiss, workspace, name, namespace, setSelectedSubHeader }) => {
```

Comment:
No typescript? 😢

Reply:
It's a .js file originally. Maybe when we remove the feature flag, we can also make this a typescript file.